draft-ietf-babel-rfc6126bis/draft-ietf-babel-rfc6126bis.xml

```diff
@@ -703,7 +710,9 @@ decreased at any time; it SHOULD NOT be increased, except immediately
 before sending a Hello packet.  (Equivalently, a node SHOULD send an
 unscheduled Hello immediately after increasing its Hello interval.)</t>
 
-<t>How to deal with received Hello TLVs and what statistics to maintain
+<t>A node that receives a Hello TLV from an unknown neighbour creates a new
+entry in the neighbour table and sets its txcost to infinity. How to deal
```

Comment:
This also constrains the protocol, what if an implementation had a way to measure txcost thanks to link-layer information?